docs/reference/index.md

```diff
@@ -1,15 +1,16 @@
 <!--[metadata]>
 +++
-title = "Compose CLI reference"
+title = "Command line reference"
 description = "Compose CLI reference"
 keywords = ["fig, composition, compose, docker, orchestration, cli,  reference"]
 [menu.main]
 identifier = "smn_compose_cli"
-parent = "smn_compose_ref"
+parent = "workw_compose"
+weight=80
 +++
 <![end-metadata]-->
 
-## Compose CLI reference
+## Compose Command line reference
```

Comment:
Thanks @shin- thanks for the comments.  

Just as an FYI to reviewers, we are going to be doing redirects from `docs.docker.com/compose` (which is index.md and the menu label) to `docs.docker.com/compose/overview/` (which shows on the menu as the overview and as `overview.md` in the source).  This is needed until my RFD on the docs redesign is done.